packages/core/src/sessionInfo/ISessionInfo.ts

```diff
@@ -39,3 +39,15 @@ export default interface ISessionInfo {
    */
   sessionId: string;
 }
+
+export interface ISessionInternalInfo {
```

Comment:
This interface could maybe use a comment to explain why it's needed, and why it's explicitly for internal use only...?